rfcs/055-prismic-api/articles.md

```diff
@@ -0,0 +1,114 @@
+        "32:15": {
+            "dimensions": { 
+                "width": 3200, 
+                "height": 1500 
+            },
+            "alt": "Photograph of a gallery installation showing a bench with yellow cushions facing a large video projection. The projection shows the inside of a McDonalds restaurant looking down from a high viewpoint in which the restaurant floor is beginning to flood with water.",
+            "copyright": "Flooded McDonalds by Superflex | | Wellcome Collection | | | |",
+            "url": "https://images.prismic.io/wellcomecollection/35e056eb53143a449cd612828899b160cb63b3a8_ep_000832_027.jpg?auto=compress,format'
+        },
+        "16:9": {
+            "dimensions": { 
+                "width": 3200, 
+                "height": 1800 
+            },
+            "alt": "Photograph of a gallery installation showing a bench with yellow cushions facing a large video projection. The projection shows the inside of a McDonalds restaurant looking down from a high viewpoint in which the restaurant floor is beginning to flood with water.",
+            "copyright": "Flooded McDonalds by Superflex | | Wellcome Collection | | | |",
+            "url": "https://images.prismic.io/wellcomecollection/09b89fe1bcd0a3a3091bcda6b5f62e5e53d572ca_ep_000832_027.jpg?auto=compress,format'
+        },
+        square: {
+            "dimensions": { 
+                "width": 3200, 
+                "height": 3200 
+            },
+            "alt": "Photograph of a gallery installation showing a bench with yellow cushions facing a large video projection. The projection shows the inside of a McDonalds restaurant looking down from a high viewpoint in which the restaurant floor is beginning to flood with water.",
+            "copyright": "Flooded McDonalds by Superflex | | Wellcome Collection | | | |",
+            "url": "https://images.prismic.io/wellcomecollection/9ed554ac496cb5238a50674c840b40df4fc43acd_ep_000832_027.jpg?auto=compress,format'
+        }
+    },
+}
+```
+
+```
+GET /articles?query=foo&filters=bar&sort=baz&sortOrder=asc
+{
+    "type": "ResultList",
+    "results": [
+        {
+            ... // as above
+        },
+        ...
+    ],
+    "pageSize": 100,
+    "totalResults": 1000,
+    "totalPages": 10,
+    "nextPage": "/articles?query=foo&filters=bar&sort=baz&sortOrder=asc&page=2",
+}
+```
+
+## Filters
+
+- **publicationDate.from**
+  DD-MM-YYYY
+- **publicationDate.to**
+  DD-MM-YYYY
+- **contributor.agent.label**
+  Includes author, artists, photographers, etc.
+- **format.label**
+  [`Comic`, `Long read`, `Prose poem`, `Podcast`, `In pictures`, `Article`, `Photo story`, `Interview`, `Book extract`]
+
+## Sort options
+
+- relevance
+- publicationDate
+
+Default sort should be by relevance, with a fallback to id if no query is provided or where documents have the same score.
+
+## Aggregations
+
+- contributor.agent.label
+- format.label
```

Comment:
Seems wrong to me. We'll want to restrict by ID in the API, not label. Which is called `name` above in any case (assuming that's correct).

Reply:
tbh I originally thought the same thing, but [the catalogue api uses contributor.agent.label](https://api.wellcomecollection.org/catalogue/v2/works?contributor.agent.label=Columbario,%20Christopher.)... Happy to switch to ID if you think that makes more sense though?

Reply:
use `id` instead